outlook/mail/pkg/commands/sendMessage.go

```diff
@@ -1,30 +0,0 @@
-package commands
-
-import (
-	"context"
-	"fmt"
-
-	"github.com/gptscript-ai/tools/outlook/mail/pkg/client"
-	"github.com/gptscript-ai/tools/outlook/mail/pkg/global"
-	"github.com/gptscript-ai/tools/outlook/mail/pkg/graph"
-	"github.com/gptscript-ai/tools/outlook/mail/pkg/util"
-)
-
-func SendMessage(ctx context.Context, info graph.DraftInfo) error {
-	c, err := client.NewClient(global.AllScopes)
-	if err != nil {
-		return fmt.Errorf("failed to create client: %w", err)
-	}
-
-	message, err := graph.CreateDraft(ctx, c, info)
-	if err != nil {
-		return fmt.Errorf("failed to create draft: %w", err)
-	}
-
-	if err := graph.SendDraft(ctx, c, util.Deref(message.GetId())); err != nil {
-		return fmt.Errorf("failed to send message: %w", err)
-	}
-
-	fmt.Println("Message sent successfully")
-	return nil
-}
```

Comment:
The tool that used this was removed at some point, so I removed it to clean things up.